test/axelar-gateway.js

```diff
@@ -81,6 +81,7 @@ describe('Axelar Gateway', () => {
             ),
         );
 
+        await publishPackage(client, deployer, 'utils');
```

Comment:
this will change deployment scripts as well i guess?

Reply:
We needs to add `utils` package deployment into the `deploy-contract` script. Since the `axelar_gateway` package depends on the `utils` package, we need to deploy it before the gateway contract in the test-sui workflow as well.